.github/workflows/snyk.yml

```
@@ -0,0 +1,67 @@
+name: "Snyk"
+
+on:
+  # Please uncomment the push event for testing purposes
+  #push:
+  #  branches: [main]
+  # Run the job at the end of every day
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  # Please check how to create an access token here https://docs.snyk.io/tutorials/amazon-web-services/aws-code-suite/snyk-security/create-account-and-obtain-a-token
+  SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+  DEFAULT_JDK_VERSION: 11
+
+jobs:
+  quarkus:
+    name: Quarkus
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "${{ env.DEFAULT_JDK_VERSION }}"
```

To speed up this workflow you could [cache the Maven dependencies](https://github.com/actions/setup-java#caching-maven-dependencies).
```suggestion
          java-version: "${{ env.DEFAULT_JDK_VERSION }}"
          cache: maven
```